superset/assets/spec/javascripts/explore/components/AdhocMetricEditPopover_spec.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import sinon from 'sinon';
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
```

Comment:
do you really need `mount` or do you know why it is required now but not previously? It's good to avoid that when possible. wondering if calling `wrapper.update()` would fix (tho the docs say that applies to just mounted components)